src/db/migration/migrate20220729-FilesShareWithMeFolder.pl

```diff
@@ -20,42 +20,47 @@
 #
 # Add Files Shared With me system folder (id = 20) to each mailbox.
 #
-
 use strict;
 use Migrate;
 
-sub bumpUpMailboxChangeCheckpoints();
```

Comment:
@gabhinavTI was this script really complete earlier? I don't see code to bump changepoint